doc/evaluation.rst

```diff
@@ -206,25 +206,55 @@ and :math:`\sigma_T^2` is estimated as:
 
-   :math:`MSE(T|M) = \displaystyle\frac{\sum_{i=1}^{N}{c_i(\hat{H_i} - M_i)^2} - N\sigma_{e}^2}{N_1+2N_2}`
+In this case variance of errors is computed as pooled variance estimator.
 
-   :math:`\sigma_T^2=\displaystyle\frac{\sum_{i=1}^{N}{c_i(\hat{H}_i - \bar{\hat{H}})^2}-(N-1)\sigma_{e}^2}{(N-1) + \displaystyle\frac{N_2(N_1+2N_2-2)}{N_1+2N_2}}`
+We first calculate the within within-subject variance of human ratings for each response, :math:`V_i`, using denominator N-1:
 
-   where 
+:math:`V_{i} = \frac{\sum_{j=1}^c H_{i,j} - \bar{H}_i}{c_i-1}`
 
-   * :math:`C_i=1` or 2 depending on the number of human scores observed for individual 𝑖.
+where
 
-   * :math:`\hat{H}` is the average of two human scores :math:`\hat{H_i} = \frac{{H_i}+{H2_i}}{2}` when two scores are available or :math:`\hat{H_i} = H_i` when only one score is available. 
+* :math:`H_{i,j}` is human score assigned by rater :math:`j` to response :math:`i`
 
-   * :math:`N_1` is the number of responses with only one human score available (:math:`N_1+N_2=N`)
+* :math:`c_i` is the total number of human scores available for response :math:`i`. For double-scored responses this equals 2. 
+
+* :math:`\bar{H}_i` is the average human rating for response :math:`i`. 
+
+We then take a weighted average of those within-responses variances:
+
+:math:`\sigma_{e}^2 = \frac{\sum_{i=1}^N V_{i} * (c_i-1)}{N}`
+
+The **true score variance** :math:`\sigma_T^2` is then estimated as 
+
+:math:`\sigma_T^2 = \frac{\sum_{i=1}^N c_i (\bar{H}_i - \bar{H})^2 -
+(N-1) \sigma_{e}^2}{c_\cdot - \frac{\sum_{i=1}^N
+c_i^2}{c_\cdot}}`
+
+where 
+
+* :math:`c_\cdot = \sum_{i=1}^N c_i` is the total number of observed human scores. 
+
+* :math:`\bar{H}_i` is the average human rating for response :math:`i`. For responses with only one rating this will be the single human score `H`.
+
+**Mean squared error** :math:`MSE(T|M)` is estimated as: 
+
+:math:`MSE(T|M) = \frac{1}{c_\cdot} \left (\sum_{i=1}^N c_i (\bar{H}_i - M_i)^2  -
+N\sigma_{e}^2 \right )`
+
+
+The formulas are derived to ensure consistent results regardless of the number of raters and the number of ratings availvable for each response. 
+
+PRMSE is computed using :ref:`rsmtool.utils.prmse_true <prmse_api>`.
+
+
+In some cases, it may be appropriate to compute variance of human errors using a different sample than the one used for main evaluations. This can be accomplished using :ref:`rsmtool.utils.variance_of_errors <ve_api>` and using an optional configuration field `rater_error_variance` in :ref:`rsmtool<rater_error_variance_rsmtool>` or :ref:`rsmtool<rater_error_variance_rsmeval>`
```

Comment:
How about: 
```In some cases, it may be more appropriate to compute the variance of human errors using a different sample than the one used for main evaluations. This can be accomplished either by using the  optional configuration field `rater_error_variance` for :ref:`rsmtool<rater_error_variance_rsmtool>` and :ref:`rsmtool<rater_error_variance_rsmeval>` or by supplying an optional parameter to :ref:`rsmtool.utils.prmse_true <prmse_api>`. To compute rater error variance on external data, you can use :ref:`rsmtool.utils.variance_of_errors <ve_api>` function.```

Reply:
Looks good with just a few minor corrections:
- " ... by supplying an optional parameter to **the** :ref:`rsmtool.utils.prmse_true <prmse_api>` **function**"
- " ... you can use _the_ :ref:`rsmtool.utils.variance_of_errors <ve_api>` function."